requirements/connectors/kdbai.txt

```diff
@@ -6,7 +6,7 @@ charset-normalizer==3.3.2
     # via requests
 idna==3.10
     # via requests
-kdbai-client==1.3.0
+kdbai-client==1.4.0
```

Comment:
If this code is specific to this version, let's also add a `kdbai-client>=1..4.0` to the associated `.in` file.